index.js

```diff
@@ -80,7 +81,10 @@ module.exports = function upsertMany(schema) {
         }
       });
 
+    //Retrieve if bulkWrite should be a ordered or unordered write
+    let {ordered} = config;
```

Comment:
Can you use `const` instead of `let`, as this value will not change, and can you move it up to line 32 where all the config is read (e.g. `const {type} = config;`)